### ISSUE TITLE
Kingdom Hearts Birth By Sleep Final Mix fade transitions look bad

### ISSUE THREAD

**Issue by TheParagone** [hrydgard/ppsspp#18912]
### Game or games this happens in

ULJM05775

### What area of the game

Land of Departure (first area) during some cutscenes. It only happens when a fade transition would normally happen. Probably a missing readback.

### What happens

When a fade transition happens instead of fading normally the game transitions from a random picture (menu, etc) to the scene happening on screen. Here's a video showing the [issue](https://youtu.be/z4YjnkszOW4).

### What should happen

The transition should happen like a normal fade transition.

### Platform

Android

### Mobile device model or graphics card (GPU)

Realme GT2 

### PPSSPP version affected

1.17.1

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the glitch (especially speed hacks or enhancements/replacements.)
- [X] Include logs or screenshots of issue.

**Comment by IskaVanSchnee** [comment]
there are 2 possible ways to fix this : 
1. change psp model to psp-1000
2. play it with the old build of ppsspp for example "ppsspp v1.11.3" funny is ppsspp starting to this version and lower are alot faster from v1.12 to the latest.

**Comment by hrydgard** [comment]
Curious that the psp version matters. We could make a compat.ini option forcing it, but I wonder why.. Does it actually somehow use the extra memory, for some extra graphical effect or something? It shouldn't be able to ..

**Comment by IskaVanSchnee** [comment]
> Curious that the psp version matters. We could make a compat.ini option forcing it, but I wonder why.. Does it actually somehow use the extra memory, for some extra graphical effect or something? It shouldn't be able to ..

well im not sure, but i'll be making a video about the performance difference soon

**Comment by TheParagone** [comment]
> there are 2 possible ways to fix this :
> 
> 1. change psp model to psp-1000
> 2. play it with the old build of ppsspp for example "ppsspp v1.11.3" funny is ppsspp starting to this version and lower are alot faster than from v1.12 to the latest.

Yep, using PSP1000 worked. Since that fix doesn't work for other games I've had similar issues with like Stein's Gate then it must be something else. Thanks for providing a fix though.
